processes/hackathon_sponsorship_process.md

```diff
@@ -0,0 +1,98 @@
+# Hackathon Sponsorship Process
+
+Process for working with a hackathon that reaches out to us for sponsorship.
+
+**Table of Contents**
+
+- [Hackathon Sponsorship Process](#hackathon-sponsorship-process)
+  - [Goals Of Sponsorship For Hack Club](#goals-of-sponsorship-for-hack-club)
+  - [Fields](#fields)
+  - [Stages](#stages)
+    - [Contacted Us](#contacted-us)
+    - [Responded](#responded)
+    - [Invited To Event](#invited-to-event)
+    - [Successful Hackathon](#successful-hackathon)
+    - [Rejected](#rejected)
+
+Hackathon sponsorships should be tracked in the `Hackathons` Streak pipeline. Just like any other Streak pipeline, make sure to link new emails to related Streak Boxes and keep the pipeline up to date when things change.
+
+## Goals Of Sponsorship For Hack Club
+
+We're looking to help spread hacker ethos and technical knowledge at these events.
```

Comment:
This is a good high level goal, but I don't think it's going to help when deciding whether or not we should attend an event.